packages/ember/src/components/mktg/promo.gts

```diff
@@ -0,0 +1,39 @@
+import type { TOC } from '@ember/component/template-only';
+
+interface PromoSignature {
+  Element: HTMLDivElement;
+  Args: {
+    productName?: string;
+    headerText?: string;
+    imageSrc?: string;
+    imageAltText?: string;
```

Comment:
After some thinking, I've decided these may be better as a yielded block so there is more room to change sizing and other things